tutorials/shaders/shader_reference/spatial_shader.rst

```diff
@@ -278,6 +271,14 @@ Below are examples of common variables calculated using the built-ins:
 +----------------------------------------+--------------------------------------------------------------------------------------------------+
 | in mat4 **INV_PROJECTION_MATRIX**      | Clip space to view space transform.                                                              |
 +----------------------------------------+--------------------------------------------------------------------------------------------------+
+| in vec3 **NODE_POSITION_WORLD**        | Node world space position                                                                        |
++----------------------------------------+--------------------------------------------------------------------------------------------------+
+| in vec3 **NODE_POSITION_VIEW**         | Node view space position                                                                         |
++----------------------------------------+--------------------------------------------------------------------------------------------------+
+| in vec3 **CAMERA_POSITION_WORLD**      | Camera world space position                                                                      |
++----------------------------------------+--------------------------------------------------------------------------------------------------+
+| in vec3 **CAMERA_DIRECTION_WORLD**     | Camera world space direction                                                                     |                                                                                                  |
```

Comment:
```suggestion
| in vec3 **CAMERA_DIRECTION_WORLD**     | Camera world space direction                                                                     |
```
There is stray whitespace and a pipe on line 280, which makes the table malformed. Everything else looks good!

Reply:
Okay I think I resolved it. It seems to render correctly now :D